packages/admin-panel/src/editor/Editor.js

```diff
@@ -5,10 +5,11 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import { InputGroup } from '@tupaia/ui-components';
 import { InputField } from '../widgets';
 import { checkVisibilityCriteriaAreMet, labelToId } from '../utilities';
 
-export const Editor = ({ fields, recordData, onEditField }) => {
+export const Editor = ({ fields, recordData, onEditField, sections }) => {
```

Comment:
I think we could probably simplify this a fair bit by just defining sections within the `fields` array? That way this whole function could be simplified to just checking for `type: 'subsection'` and mapping those to `InputGroup`, and it'd still maintain backwards compatibility.

```
fields = [
  {
    type: 'subsection', // Or whatever language you think suits
    title: 'foo',
    description: 'bar',
    fields: [
      {
        // In theory we could eventually support nested sections but that's def beyond the scope of this ticket
        Header: 'Name',
        source: 'name',
      }
    ]
  }
]
```

This way the structure of the data matches the way it's being displayed which simplifies the dev process. It also means we don't have to update both arrays when adding/moving a field, which should completely remove the chance of data mismatches.